src/Query.h

```diff
@@ -31,7 +31,9 @@ public slots:
     void requestNextRowSet(int rowCount);
 
 private:
-
+    QList<int> m_numberTypeIds;
```

Comment:
`QSet` might be better, or even a `QHash<Integer /*typeId*/, QColor> m_foregroundColor;` if you want to have fewer collections.

Reply:
whatever is going to be easier to link to Settings and color saved there (whenever it gets implemented).